app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+
+import { Formik, Form } from "formik";
+import { Check } from "neetoicons";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import { CONTACT_FORM_VALIDATION_SCHEMA, ROLES_DATA } from "../constants";
+
+const ContactForm = ({ onClose, contact, isEdit }) => {
```

Comment:
https://github.com/bigbinary/bigbinary-website/issues/2462

Reply:
Is it okay to keep the file name as ContactForm.jsx? Here, I have followed similar naming convention as in the Notes Pane folder, which was initially present in the repo.